config/routes.rb

```diff
@@ -5,7 +5,8 @@
 
   namespace :api, defaults: { format: :json } do
     namespace :v1 do
```

Comment:
Are there other common alternatives? This is what we were taught at Turing, but I have no particular attachment to it.